### DISCUSSION TITLE
Please help me understand error message and how to fix it!

### DISCUSSION THREAD

**Discussion by LostFrog87** [Monika-After-Story/MonikaModDev#8451] [QA - tech support + general] [answered]
Help the game won’t open and this error message is popping up!
![CD18EF33-06BA-44A3-9F53-0C64D2622E2E](https://user-images.githubusercontent.com/95949465/145661645-aa335f8f-e492-44a9-82ca-5da7c818cc48.jpeg)

**Comment by AshtonSnapp** [comment]
The pertinent line is this one:

`IOError: [Errno 2] No usable temporary directory found in ['/var/folders/3h/mwmbhiz97d16_b0p2nbgtlk40000gn/T/', '/tmp', '/var/tmp', '/usr/tmp', '/Users/owner/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club']`

In short, the game was trying to make a temporary file in one of those listed directories, but found that none of the directories were actually usable for putting temporary files in. Yeah, I don't really understand it either. But that's what it says, so that's what happened.

**Comment by LostFrog87** [comment]
> The pertinent line is this one:
> 
> `IOError: [Errno 2] No usable temporary directory found in ['/var/folders/3h/mwmbhiz97d16_b0p2nbgtlk40000gn/T/', '/tmp', '/var/tmp', '/usr/tmp', '/Users/owner/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club']`
> 
> In short, the game was trying to make a temporary file in one of those listed directories, but found that none of the directories were actually usable for putting temporary files in. Yeah, I don't really understand it either. But that's what it says, so that's what happened.

Is there anyway I can fix it?

**Comment by Booplicate** [comment, marked as answer]
You disk is either full, or you (or the game) don't have permissions to create a temp directory. So check your disk space and share permissions.
